php-frontend/src/main/java/org/sonar/php/tree/impl/declaration/ClassDeclarationTreeImpl.java

```diff
@@ -77,10 +79,23 @@ protected ClassDeclarationTreeImpl(
     this.closeCurlyBraceToken = closeCurlyBraceToken;
   }
 
+  /**
+   * @deprecated since 3.25 - Use {@link #modifiersToken()} instead.
```

Comment:
You have to name the SonarQube version.
```suggestion
   * @deprecated since SonarQube 9.7 - Use {@link #modifiersToken()} instead.
```